core/frontend/package.json

```diff
@@ -74,6 +74,7 @@
     "@types/semver": "7.3.10",
     "@types/superagent": "^4.1.14",
     "@types/sinon": "^9.0.0",
+    "babel-loader": "~8.2.5",
```

Comment:
dont you need to also add a dep on babel-plugin-instanbul to actually use this?
same applies for all pkg.json where you only added babel-loader

Reply:
I can, but the reason I didn't was the old instanbul plugin was not in every package... and just replacing seemed to work, but yeah makes sense.

Reply:
i blame not enabling strict deps in wush w/ pnpm.
but also looks like we haven't been running instrumented tests since 3.0-dev.xyz so we wouldve never actually ran into an issue where trying to require a dep that didnt exist

Reply:
That is correct, but I believe I ran them with current deps unchanged. At any rate, no problem